containerapps/azure.yaml

```diff
@@ -0,0 +1,117 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: azure-search-openai-demo
+metadata:
+  template: azure-search-openai-demo@0.0.2-beta
+services:
+  backend:
+    project: ../app/backend
+    language: py
+    host: containerapp
+    hooks:
+      prebuild:
+        windows:
+          shell: pwsh
+          run:  cd ../frontend;npm install;npm run build
+          interactive: false
+          continueOnError: false
+        posix:
+          shell: sh
+          run:  cd ../frontend;npm install;npm run build
+          interactive: false
+          continueOnError: false
+infra:
+  provider: bicep
+  path: ../infra
+
+pipeline:
+  variables:
```

Comment:
Given that there's now another azure.yaml, we need to make sure folks update both azure.yaml's whenever there's a new environment variable. Can you add a section to CONTRIBUTING.md about "Adding new azd environment variables" that points to both azure.yamls, GitHub workflows, and ADO pipeline.
Otherwise I'm worried we'll forget about ACA.

Reply:
Yeah, I didn't update these files...

Reply:
Do we need to update GitHub workflows, and ADO pipeline files to allow deploying to aca too?

Reply:
Where do you think I should place the caveat?